Makefile

```diff
@@ -5,21 +5,46 @@ generated = myhoard/version.py
 all: $(generated)
 
 PYTHON ?= python3
-PYTHON_SOURCE_DIRS = myhoard/
-PYTHON_TEST_DIRS = test/
-ALL_PYTHON_DIRS = $(PYTHON_SOURCE_DIRS) $(PYTHON_TEST_DIRS)
```

Comment:
fmt and pylint is no longer executed on test directories. At least formatting should be consistent also in tests.

Reply:
`PYTHON_SOURCE_DIRS` has been consolidated so it includes `test`. `fmt` and `pylint` are called against both.

Reply:
Oh yeah could've actually read the value, just assumed it didn't based on the name...